.gitignore

```diff
@@ -9,4 +9,5 @@
 *.lk
 *.int
 *.backup
-*.int
\ No newline at end of file
+*.int
+
```

Comment:
ill fix it tomorrow it was a change to this file that I didnt mean to have commit I removed it but git didnt think I removed the line...